service/src/test/java/uk/gov/hmcts/reform/fpl/service/removeorder/OrderRemovalActionsTest.java

```diff
@@ -6,58 +6,64 @@
+import uk.gov.hmcts.reform.fpl.model.StandardDirectionOrder;
 import uk.gov.hmcts.reform.fpl.model.order.CaseManagementOrder;
 import uk.gov.hmcts.reform.fpl.model.order.generated.GeneratedOrder;
 
-import java.util.UUID;
-
 import static java.lang.String.format;
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
 class OrderRemovalActionsTest {
     @Mock
-    private CMOOrderRemovalAction cmoOrderRemovalAction;
+    private CMORemovalAction cmoOrderRemovalAction;
 
     @Mock
     private GeneratedOrderRemovalAction generatedOrderRemovalAction;
 
+    @Mock
+    private SDORemovalAction sdoRemovalAction;
+
     @InjectMocks
     private OrderRemovalActions orderRemovalActions;
 
-    private static final UUID ORDER_ID = UUID.randomUUID();
     private static final CaseManagementOrder CASE_MANAGEMENT_ORDER = CaseManagementOrder.builder().build();
     private static final GeneratedOrder GENERATED_ORDER = GeneratedOrder.builder().build();
+    private static final StandardDirectionOrder STANDARD_DIRECTION_ORDER = StandardDirectionOrder.builder().build();
 
     @Test
     void shouldReturnGeneratedOrderActionWhenGettingActionForGeneratedOrder() {
         when(cmoOrderRemovalAction.isAccepted(CASE_MANAGEMENT_ORDER)).thenReturn(true);
 
-        assertThat(orderRemovalActions.getAction(ORDER_ID, CASE_MANAGEMENT_ORDER))
+        assertThat(orderRemovalActions.getAction(CASE_MANAGEMENT_ORDER))
             .isEqualTo(cmoOrderRemovalAction);
     }
 
     @Test
     void shouldReturnCMOActionWhenGettingActionForCMO() {
         when(generatedOrderRemovalAction.isAccepted(GENERATED_ORDER)).thenReturn(true);
 
-        assertThat(orderRemovalActions.getAction(ORDER_ID, GENERATED_ORDER))
+        assertThat(orderRemovalActions.getAction(GENERATED_ORDER))
             .isEqualTo(generatedOrderRemovalAction);
     }
 
+    @Test
+    void shouldReturnSDOActionWhenGettingActionForSDO() {
+        when(sdoRemovalAction.isAccepted(STANDARD_DIRECTION_ORDER)).thenReturn(true);
+
+        assertThat(orderRemovalActions.getAction(STANDARD_DIRECTION_ORDER))
+            .isEqualTo(sdoRemovalAction);
+    }
+
     @Test
     void shouldThrowAnExceptionWhenActionTypeFailsToMatchToRemovableOrder() {
         when(cmoOrderRemovalAction.isAccepted(GENERATED_ORDER)).thenReturn(false);
 
-        RemovableOrderActionNotFoundException actualException
-            = assertThrows(RemovableOrderActionNotFoundException.class,
-            () -> orderRemovalActions.getAction(ORDER_ID, GENERATED_ORDER)
-        );
-
-        assertThat(actualException.getMessage()).isEqualTo(
-            format("Removable order action not found for order %s", ORDER_ID)
-        );
+        assertThatThrownBy(() -> orderRemovalActions.getAction(GENERATED_ORDER))
```

Comment:
I think the previous test was more inline. You can create a mock order that for all returns false for all of the actions to obtain this behaviour